apps/blockchain/lib/blockchain/account/repo.ex

```diff
@@ -188,14 +192,20 @@ defmodule Blockchain.Account.Repo do
         {status, account, _} = account_from_cache(updated_repo.cache, address)
 
         updated_cache =
-          Cache.update_account(updated_repo.cache, address, {status, account, value_to_cache})
+          Cache.update_account(
+            updated_repo.cache,
+            address,
+            {status, account, {:clean, value_to_cache}}
+          )
 
         repo_with_cached_code = %{updated_repo | cache: updated_cache}
 
         {repo_with_cached_code, found_code}
 
       true ->
-        {updated_repo, {:ok, code}}
+        {_status, code} = code
```

Comment:
this `cond` is getting rather big, any plans to start breaking it apart?

Reply:
I moved code caching to separate functions